src/classes/CW_AudienceCSVProcessHelper.cls

```diff
@@ -0,0 +1,406 @@
+/**
+* File Name :   CW_AudienceCSVProcessHelper.cls
+* Description : Helper Class for processing iported CSV file in batches
+*               
+* Modification Log
+* ============================================================================
+* Ver Date       Author         Modification
+* --- ---------- -------------- --------------------------
+* 0.1 20/05/2014 Alex PAnka     Initial version
+*
+*
+*/
+public with sharing class CW_AudienceCSVProcessHelper {
```

Comment:
Classes and triggers should use the `UpperCamelCase` naming convention. I suggest to rename accordingly.